testing/test_assertrewrite.py

```diff
@@ -1708,3 +1712,47 @@ def test_foo():
             cache_tag=sys.implementation.cache_tag
         )
         assert bar_init_pyc.is_file()
+
+
+class TestReprSizeVerbosity:
+    """
+    Check that verbosity also controls the string length threshold to shorten it using
+    ellipsis.
+    """
+
+    @pytest.mark.parametrize(
+        "verbose, expected_size", [(0, 240), (1, 240), (2, 240), (3, 2400), (4, None)]
```

Comment:
Since `_DEFAULT_REPR_MAX_SIZE` is already imported, would use it instead of 240 here.